common/hogvm/__tests__/__snapshots__/stl.js

```diff
@@ -61,7 +61,7 @@ function toIntervalMonth(val) { return __toHogInterval(val, 'month') }
 function toIntervalDay(val) { return __toHogInterval(val, 'day') }
 function toDateTime (input, zone) { return __toDateTime(input, zone) }
 function toDate (input) { return __toDate(input) }
-function substring(s, start, length) {
+function substring(s, start, length = s.length - start + 1) {
     if (typeof s !== 'string') return '';
```

Comment:
I fear this default parameter may will crash on `s.length` if `s` is not a string

Reply:
Moved the fallback length behind the string check